### PR TITLE
Checkbox, Switch: add Android & iOS docs pages

### PR DESCRIPTION
**Pull request by dangerismycat**
Checkbox iOS
https://jira.pinadmin.com/browse/GESTALT-5092
https://coda.io/d/Gestalt-Team-Folder_d2LeXkQ1kVX/Checkbox_sudFS

Checkbox Android
https://jira.pinadmin.com/browse/GESTALT-5093
https://coda.io/d/Gestalt-Team-Folder_d2LeXkQ1kVX/Checkbox_sudFS

Switch iOS
https://jira.pinadmin.com/browse/GESTALT-5094
https://coda.io/d/Gestalt-Team-Folder_d2LeXkQ1kVX/Switch_suGx6#_lu-L9

Switch Android
https://jira.pinadmin.com/browse/GESTALT-5095
https://coda.io/d/Gestalt-Team-Folder_d2LeXkQ1kVX/Switch_suGx6#_lu-L9

### CODE REVIEW

File: docs/markdown/android/switch.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/c0/4c/97/c04c970f0d926842cf4ef89ad42b9347.jpg" noPadding alt="Example of disabled inactive state of Switch" />
+    **Disabled | Inactive state**
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/80/29/45/802945dd6c7a4dcf2eb43bc4d1cc9f3f.jpg" noPadding alt="Example of default active state of Switch" />
+    **Default | Active state**
+  </Group>
```
Comment (ayeshakmaz) [inline docs/markdown/android/switch.md:114]: I would reverse these two, I'd expect both active states first (also matches how it's listed above)

File: docs/markdown/android/checkbox.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+### State
+
+1. **Unchecked [Default]**
+1. **Checked**
+1. **Error**
+1. **Indeterminate**
+1. **Disabled**
+
+<ThreeCol>
```
Comment (ayeshakmaz) [inline docs/markdown/android/checkbox.md:96]: Is there a way to add space between these? they're kind of cramped 
Screen Shot 2022-12-15 at 10 24 18 PM

File: docs/markdown/android/checkbox.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    <ImgContainer src="https://i.pinimg.com/originals/e7/07/02/e70702e40416c5f4f89e92eca2bbb490.jpg" noPadding alt="Example of incorrect immediate effect use" />
+    <Dont title="Don't" />
+    Use a Checkbox to turn a state on and off with immediate effect. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple and Android’s accessibility features, such as VoiceOver and TalkBack to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple and Android’s accessibility documentation for more information:
+<TwoCol>
```
Comment (ayeshakmaz) [inline docs/markdown/android/checkbox.md:60]: We don't usually list the iOS accessibility info on Android pages, is this a new practice?

File: docs/markdown/android/switch.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+</TwoCol>
+
+## Accessibility
+
+Switches should have [Labels](https://gestalt.pinterest.systems/web/label) that can be read by screen readers, and that can be tapped to make it easier for users to select and deselect.
+People use Apple and Android’s accessibility features, such as VoiceOver and TalkBack to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple and Android’s accessibility documentation for more information:
+<TwoCol>
+  <Group>
+    [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/)
```
Comment (ayeshakmaz) [inline docs/markdown/android/switch.md:59]: Ditto here

**Review by ayeshakmaz — APPROVED** [review]

File: docs/markdown/android/checkbox.md (hunk 3 above)
Comment (dangerismycat) [reply docs/markdown/android/checkbox.md:60]: Not sure — I think Kate and Hector are rethinking how to handle mobile docs given the large amount of overlap/duplication. This is how it was in the doc so I'll leave as-is for now and we can address in the future — either removing references to the platform that doesn't belong, or further merging mobile pages.

**Pull request merged**